Orange/widgets/data/owtranspose.py

```diff
@@ -79,11 +79,9 @@ def apply(self):
         self.clear_messages()
         transposed = None
         if self.data:
-            options = dict()
-            if self.feature_type:
-                options["feature_names_column"] = self.feature_names_column
             try:
-                transposed = Table.transpose(self.data, **options)
+                transposed = Table.transpose(
+                    self.data, feature_names_column=self.feature_names_column)
```

Comment:
This is a bit wrong, but I wouldn't like to `push -f` over it, and lose the comments I made above. Please change this to .... `feature_names_column = self.feature_type and self.feature_names_column`.